packages/features/ee/billing/credit-service.ts

```diff
@@ -127,6 +127,9 @@ export class CreditService {
       });
   }
 
+  /*
+    also returns true if team has no available credits but limitReachedAt is not yet set
```

Comment:
As long as limitReachedAt is not set, we keep allowing SMS sending. After an SMS is successfully sent, we charge the team and then set limitReachedAt if they are out of credits.